mithril-common/src/cardano_transactions_preloader.rs

```diff
@@ -105,8 +148,19 @@ mod tests {
         }
     }
 
+    struct CardanoTransactionsPreloaderActivationWithError {}
```

Comment:
This could probably have been mocked instead?

Reply:
Done